deegree-datastores/deegree-featurestores/deegree-featurestore-shape/src/main/java/org/deegree/feature/persistence/shape/DbfIndexImporter.java

```diff
@@ -304,7 +303,7 @@ ConnectionProvider createIndex()
 
         connProvider = new LegacyConnectionProvider( "jdbc:h2:" + dbfile, "SA", "", false, null );
 
-        if ( new File( dbfile.toString() + ".h2.db" ).exists() ) {
+        if ( new File( dbfile.toString() + ".mv.db" ).exists() ) {
```

Comment:
The file naming strategy has been changed at h2.
This change leads to the fact that the detection of whether the table already exists fails because currently the existence of the table is assumed to be the same as the existence of the file.